src/test/java/com/google/firebase/remoteconfig/TemplateTest.java

```diff
@@ -49,11 +49,13 @@
           "greeting_header", new Parameter()
                   .setDefaultValue(ParameterValue.inAppDefault())
                   .setDescription("greeting header text")
-                  .setConditionalValues(CONDITIONAL_VALUES),
+                  .setConditionalValues(CONDITIONAL_VALUES)
+                  .setValueType(ParameterValueType.STRING),
           "greeting_text", new Parameter()
                   .setDefaultValue(ParameterValue.inAppDefault())
                   .setDescription("greeting text")
                   .setConditionalValues(CONDITIONAL_VALUES)
+                  .setValueType(ParameterValueType.STRING)
```

Comment:
Do we have coverage here for parameters without types?

Reply:
Good call. Updated to have a parameter without a type.